Tests/Expected/Images/allvalues-context-defaults-customname.swift

```diff
@@ -10,8 +10,10 @@
 
 // swiftlint:disable file_length
 // swiftlint:disable line_length
-
+// swiftlint:disable nesting
```

Comment:
I don't agree adding this rule just to "unify" the templates. This template doesn't have nesting, only the dot-syntax variants.